Tests/BasicsTests/Archiver/ZipArchiverTests.swift

```diff
@@ -158,117 +158,3 @@ class ZipArchiverTests: XCTestCase {
          }
      }
 }
-
-class ArchiverTests: XCTestCase {
```

Comment:
This test exercises no production code. It creates a `MockArchiver` implementation which is broken if you call cancel before an action is called. It also makes it seem pretty clear that when spm moves to async/await it is going to need to move to async/await cancelation and ditch the Cancellable protocol because of race conditions around ordering of concurrent operations like do X and cancel X

Reply:
yes, similar to what we had done in service lifecycle, we would need to use different cancelation strategy as we move to swift concurrency

Reply:
sounds good. I just wanted to explain I was deleting a "test" instead of updating/fixing it.